### PR TITLE
add public persona

### PR DESCRIPTION
**Pull request by killerninja8**

### CODE REVIEW

**Comment by killerninja8** [comment]
research question and user persona section is now complete.

File: reports/m1_proposal.md [hunk 1]
```diff
@@ -24,6 +24,30 @@ We will use these columns to derive new variables, such as an [Air Quality Index
 
 ## Research Questions
 
+Dr. Hu is a public health official in the Ministry of Health. She wishes to monitor public health outcomes, advise on health policy, and initiate public health programs. Dr. Hu is not a data science professional but she has immense knowledge of epidemiology, medicine, and public health. 
+
+Dr. Hu wishes to understand the impact of air pollution on public health and identify areas where government intervention can reduce negative health outcomes. She plans to use this dashboard to identify problem areas, recommend specific courses of action for the public health ministry, and deliver new health campaigns tailored to the needs of citizens to mitigate the ill-effects of air pollution. 
+
+Through the Pollution Tracker dashboard, Dr. Hu aims to:
+
+* [Explore] global and local air pollution trends over several years to gain an in-depth understanding of pollution patterns and their evolution over time.
```
Comment (meretelutz) [inline reports/m1_proposal.md:33]:
```suggestion
* Explore global and local air pollution trends over several years to gain an in-depth understanding of pollution patterns and their evolution over time.
```

File: reports/m1_proposal.md [hunk 2]
```diff
@@ -24,6 +24,30 @@ We will use these columns to derive new variables, such as an [Air Quality Index
...
+* [Compare] levels of key pollutants (PM, CO, Ozone, NOx, SOx) across various regions to pinpoint differences and similarities in air quality.
```
Comment (meretelutz) [inline reports/m1_proposal.md:34]:
```suggestion
* Compare levels of key pollutants (PM, CO, Ozone, NOx, SOx) across various regions to pinpoint differences and similarities in air quality.
```

File: reports/m1_proposal.md [hunk 3]
```diff
@@ -24,6 +24,30 @@ We will use these columns to derive new variables, such as an [Air Quality Index
...
+* [Identify] pollution hotspots where urgent action is needed, and highlight times when pollution levels rise significantly, to sound the alarm for potential air quality crises.
```
Comment (meretelutz) [inline reports/m1_proposal.md:35]:
```suggestion
* Identify pollution hotspots where urgent action is needed, and highlight times when pollution levels rise significantly, to sound the alarm for potential air quality crises.
```

File: reports/m1_proposal.md [hunk 4]
```diff
@@ -24,6 +24,30 @@ We will use these columns to derive new variables, such as an [Air Quality Index
...
+* [Derive] actionable insights by analyzing pollution levels, trends, rates of change, and anomalies. These insights will inform public health advisories, guide health policy formulation, and mitigate air pollution health impacts for a healthier and happier population.
```
Comment (meretelutz) [inline reports/m1_proposal.md:36]:
```suggestion
* Derive actionable insights by analyzing pollution levels, trends, rates of change, and anomalies. These insights will inform public health advisories, guide health policy formulation, and mitigate air pollution health impacts for a healthier and happier population.
```

**Review by meretelutz — CHANGES_REQUESTED** [review]
Great job! Really like the usage scenario you came up with. I just had one small issue with the formatting of a list, so I suggested some changes.

**Comment by killerninja8** [comment]
Thanks for the suggestions @meretelutz! I've committed those changes

**Pull request merged**